packages/sdk-middleware-auth/test/build-requests.spec.js

```diff
@@ -31,6 +31,28 @@ describe('buildRequestForClientCredentialsFlow', () => {
     })
   })
 
+  it('parses a host that ends with \'/\'', () => {
```

Comment:
I mean, you shouldn't have been able to commit since we have git hooks setup. Did you actually set them up correctly?